notes.md

```diff
@@ -0,0 +1,110 @@
+# Notes
```

Comment:
Where did those file changes come from? They are not related to the current documentation